stdlib/mem.jou

```diff
@@ -1,13 +1,49 @@
 # Memory management
 
+# TODO: repalce the long with size_t
```

Comment:
This comment doesn't belong here. Currently Jou only supports 64-bit systems, so `long` and `size_t` are always of the same size.